web-testbed/tests/widgets/test_button.py

```diff
@@ -0,0 +1,55 @@
+from pytest import approx, fixture
+from tests.assertions import assert_background_color
+from tests.data import TEXTS
+from tests.tests_backend.proxies.object_proxies import ButtonProxy, MockProxy
+
+TRANSPARENT = "transparent"
+
+
+@fixture
+async def widget():
+    return ButtonProxy("Hello")

```

Comment:
Something to investigate - can this be pushed into the backend so that it's the *backend* responsibility to create the object (e.g., something like `test_backend.factory("Button")("Hello")`). That would allow the test case to remain generic - the current test suite can be satisfied by literally instantiating `toga.Button`, but the web backend can instantiate `ButtonProxy()` (which also opens the door to removing the need for `object_proxies` (or at least moving that functionality to be clearly in the backend)